frontend_build/src/webpack.config.base.js

```diff
@@ -101,6 +101,8 @@ var config = {
       'kolibri_module': path.resolve('kolibri/core/assets/src/kolibri_module'),
       'core-constants': path.resolve('kolibri/core/assets/src/constants'),
       'core-base': path.resolve('kolibri/core/assets/src/vue/core-base'),
+      'core-actions': path.resolve('kolibri/core/assets/src/actions'),
```

Comment:
There will be some merge conflicts when bringing in master, but won't be too hard to resolve.